main.py

```diff
@@ -3,8 +1,48 @@
+import json
 from db.models import Race, Skill, Player, Guild
 
 
+def read_file() -> object:
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+    return players
+
+
 def main() -> None:
-    pass
+    players_data = read_file()
+
+    for player_name, player_data in players_data.items():
```

Comment:
Ensure that `players_data` is a dictionary. If the JSON structure changes, this line might raise an exception. Consider adding validation or error handling to ensure the expected structure.